### ISSUE TITLE
[iOS, Android] No Native Control for Subtitles showing.

### ISSUE THREAD

**Issue by nicolaosm** [TheWidlarzGroup/react-native-video#3232]
### What happened?

When `selectedTextTrack` and `textTracks` are provided, subtitles render correctly on screen, but there is no native control (button) to disable the subtitles or to choose a different subtitle track.

**Expected behavior**

A native subtitles button should be shown (e.g. on Android) when subtitles are available, allowing the user to toggle or switch subtitle tracks.

**Reproducible sample code**

```jsx
selectedTextTrack={{type: 'language', value: 'en'}}
        textTracks={[
          {
            title: 'test',
            language: 'en',
            type: TextTrackType.VTT,
            uri: 'https://bitdash-a.akamaihd.net/content/sintel/subtitles/subtitles_en.vtt',
          },
        ]}
```

### Steps to reproduce

1. Add `selectedTextTrack` and `textTracks` props to the `<Video>` component (see sample code above).
2. Play the video; subtitles show on the front end.
3. Observe that there is no native button to disable the subtitles or choose a different subtitle track.

### react-native-video version

6.0.0-alpha.7

### Platforms

iOS, Android

### OS version

iOS 16.4

### Device

Real device

### Media / source type

Other

**Comment by nicolaosm** [comment]
I also believe this should be critical for release @freeboub! Thank you

**Comment by freeboub** [comment]
@nicolaosm I agree this is an issue, but as it was not working on 5.2, this is not critical

**Comment by nicolaosm** [comment]
Yes I agree, but the whole subtitles feature is not usable if we're unable to change or disable subtitles of a video. Maybe worth prioritizing.

**Comment by TWG-AI** [comment]
Thanks for the report! This has been logged for the team.

This is a known gap — when custom `textTracks` (sidecar VTT files) are used, the native player controls UI doesn't wire them up for the CC toggle button. The subtitles themselves render, but native UI track switching isn't exposed. A similar gap exists on v7 as well.

As a workaround, you can build custom track-selection UI and control which track is shown via the `selectedTextTrack` prop programmatically.

> This is an automated triage comment — please double-check anything here. A human maintainer will follow up.
